db/migrate/20250505105238_create_invoices.rb

```diff
@@ -0,0 +1,15 @@
+class CreateInvoices < ActiveRecord::Migration[8.0]
+  def change
+    create_table :invoices do |t|
+      t.string :id
```

Comment:
The `id` should be a `uuid`, see [this](https://guides.rubyonrails.org/active_record_migrations.html#using-uuids-instead-of-ids-for-primary-keys) on creating a table with uuid pk